packages/code-editor/src/CodeDisplay.tsx

```diff
@@ -74,22 +79,21 @@ const LineContent = styled(Span)`
 `
 
 /**
- * The PrismEditor provides a useful implementation of prism-react-renderer
- * for Looker apps. Syntax highlighting is available for all supported SDK languages.
+ * Provides a view-only syntax highlighter for all supported SDK languages.
  * TODO: LookML syntax highlighting
  */
-export const CodeDisplay: FC<PrismEditorProps> = ({
+export const CodeDisplay: FC<CodeDisplayProps> = ({
   language = 'json',
   code,
   pattern = '',
+  transparent = false,
 }) => {
-  const highlighterLang = getPrismLanguage(language)
   return (
     <Highlight
       {...defaultProps}
       code={code}
-      language={highlighterLang}
-      theme={getOverridenTheme()}
+      language={getPrismLanguage(language)}
+      theme={getOverridenTheme(transparent)}
```

Comment:
I don't know what the thinking is on calling util functions like this in the markdown. Probably fine. I typically establish the values as consts (or use state if they can change from user interaction) but defer to @bryans99 or @josephaxisa on this topic

Reply:
Yeah, curious to know the standard on this.